.env.example

```
@@ -6,8 +6,10 @@ VITE_APP_ENV=local
 VITE_API_URL=http://localhost:8080/v1
 
 # Auth
+# local / self-hosted always use Flexprice backend /auth (signup + login), regardless of provider.
+# Cloud envs (development/production) default to supabase unless VITE_AUTH_PROVIDER=flexprice.
 VITE_AUTH_ENABLED=false
-VITE_AUTH_PROVIDER=supabase              # flexprice | supabase
+VITE_AUTH_PROVIDER=flexprice              # flexprice | supabase
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)